resources/assets/components/utilities/SiteFooter/SiteFooter.js

```diff
@@ -19,6 +19,7 @@ const SiteFooter = () => {
                 href="https://www.tiktok.com/@dosomething"
                 title="dosomething on Tik Tok"
                 className="hover:text-white"
+                style={{ padding: 'inherit' }}
```

Comment:
Ah looks like we're inheriting this [Forge footer styling](https://github.com/DoSomething/forge/blob/c1be19d27eb45e6151cbef7558f4b0ec41728e89/scss/_regions/_footer.scss#L175). Could we simply use `padding: 0;` here instead?
```suggestion
                style={{ padding: 0 }}
```

Reply:
yup gave `padding: 0 ` a try and it works the same 👍